src/deepsparse/loggers/prometheus_logger.py

```diff
@@ -27,24 +28,31 @@
 try:
     from prometheus_client import (
         REGISTRY,
+        CollectorRegistry,
         Summary,
         start_http_server,
         write_to_textfile,
     )
 
     prometheus_import_error = None
 except Exception as prometheus_import_err:
```

Comment:
Unrelated to this PR, but we should move this try/except to import in loggers/__init__.py instead. That way we just can set PrometheusLogger to None, and this file can safely assume prometheus is installed

Reply:
will address in a separate PR